cookbooks/van-keken-vof/doc/van-keken-vof.tex

```diff
@@ -0,0 +1,216 @@
+Instead we compute the initial volume fractions using a signed distance
+function $\phi$ as follows~\cite{JMR:2019,JMR-EGP:2019}.
+
+First we create the function $\phi$, which has the following two properties: 1) it is positive 
+in the region that contains one of the fluids, which we will refer to as fluid~1, and negative 
+in the complement of this region, which we will refer to as fluid~2, and 2) at each point in 
+the domain the magnitude $| \phi |$ of $\phi$ is the distance to the boundary between the 
+two fluids or materials.
+In the computations shown here, we use an approximation $\tilde{\phi}$ to $\phi$ such that   
+difference between $\tilde{\phi}$ and $\phi$ is small enough for the purposes of making the 
+computations high-quality.
+The primary advantage of choosing this particular initialization algorithm is that it allows us 
+to more accurately reproduce the initial condition on a sub-grid scale than would otherwise be 
+possible on the coarser grid on which we compute the time evolution of the interface. 
+
+\vspace{06pt}
+
+\lstinputlisting[language=prmfile]{cookbooks/van-keken-vof/doc/main.part.prm.out}
+
+The relevant sections of the parameter file for this type of initialization of the VOF method 
+appears immediately above.
+In particular, the combination of \texttt{Number of initialization samples} with the 
+\texttt{level set} initialization type indicates that our initialization will consist of 
+dividing each grid cell into $16 \times 16$ subcells and the distance to the given initial 
+interface $f(x)$, provided in \texttt{Function expression}, is computed in each of the 256 
+subcells.
+We then use this information to compute a piecewise linear interface approximation to $f(x)$.
+The volume fraction in each subcell is then found in the manner described 
+in~\cite{JMR:2019,JMR-EGP:2019}.
+This initialization procedure provides a much finer and thus, more accurate, initial condition 
+than the standard VOF initialization procedure described above.
+
+While the visualization configuration in a typical parameter file is sufficient for most 
+purposes, when using the VOF method one has the ability to see the division between the fluids 
+reconstructed by the VOF algorithm in each cell.
+This is accomplished by plotting the zero contour of a field $\tilde\psi$ that
+is generated to be $0$ on the reconstructed interface, positive in the region
+with fluid~1, and negative in the region with fluid~2.
+However $\tilde{\psi}$ does not satisfy the requirement that the magnitude is
+equal to the distance to the interface as would be required for the signed
+distance function $\tilde{\phi}$.
+The modifications to the parameter file that are necessary in order to draw the reconstructed 
+boundary as a contour are shown immediately below.
+The full configuration file for this version of the benchmark problem can be found at 
+\url{cookbooks/van-keken-vof.prm}.
```

Comment:
I think you're missing a directory name under `cookbooks/` here

Reply:
```suggestion
\url{cookbooks/van-keken-vof/van-keken-vof.prm}.
```
Comment addeded to ease use of github suggestion batch.